finder-app/manual-linux.sh

```diff
@@ -20,9 +20,19 @@ else
 	OUTDIR=$1
 	echo "Using passed directory ${OUTDIR} for output"
 fi
-
-mkdir -p ${OUTDIR}
-
+######################################################################
+if ! [ -d "${OUTDIR}" ]
```

Comment:
@akshithaluguri22 this shouldn't be required, mkdir -p should create if needed and should not error if existing